README.md

```diff
@@ -27,7 +27,42 @@ Using this package requires a working Go environment. [See the install instructi
 
 Go Modules are required when using this package. [See the go blog guide on using Go Modules](https://blog.golang.org/using-go-modules).
 
-### Using `v2` releases
+### Golang version 1.14 
```

Comment:
Whether or not people are using 1.14, this library only officially supports modules, which is explicitly stated already.

Reply:
But the installation instructions are different since the 1.14 changed the way modules are handled

Reply:
At a basic level, the installation instructions are still `GO111MODULE=on go get github.com/urfave/cli/v2`. Go 1.14 hasn't fundamentally changed that.